### ISSUE TITLE
[Active Learning] Create minimal active learning run with a real dataset and kNN classifier

### ISSUE THREAD

**Issue by MalteEbner** [lightly-ai/lightly#138]
Create a script to run an active learning episode. The input is not a dataset, but an embedding.csv with the soft labels being the true labels. As an example, use the clothing dataset: https://github.com/alexeygrigorev/clothing-dataset-small and embed its train and test set using the CLI.

The episode itself consists of following steps:

1. Upload either the true images, thumbnails of it or fake data.
2. Upload the `embedding.csv`
3. Perform a sampling and get the chosen filenames back.
4. Subset the dataset to the labeled_set provided by the filenames
5. use a kNN. e.g. from https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html, fit it on the labeled_set (5.1), evaluate on a separate test set (5.2), and predict_proba on the unlabeled_set (5.3) and 
6. save the predictions on the unlabeled_set in the al_scorer
Continue the loop starting from step 3.
